internal/uvm/scsi.go

```diff
@@ -330,11 +366,13 @@ func (uvm *UtilityVM) addSCSIActual(ctx context.Context, hostPath, uvmPath, atta
 // device or allocates a new one if not already present.
 // Returns the resulting *SCSIMount, a bool indicating if the scsi device was already present,
 // and error if any.
-func (uvm *UtilityVM) allocateSCSIMount(ctx context.Context, readOnly bool, hostPath, uvmPath, attachmentType string, vmAccess VMAccessType) (*SCSIMount, bool, error) {
-	// Ensure the utility VM has access
-	err := grantAccess(ctx, uvm.id, hostPath, vmAccess)
-	if err != nil {
-		return nil, false, errors.Wrapf(err, "failed to grant VM access for SCSI mount")
+func (uvm *UtilityVM) allocateSCSIMount(ctx context.Context, readOnly bool, hostPath, uvmPath, attachmentType, evdType string, vmAccess VMAccessType) (*SCSIMount, bool, error) {
+	if attachmentType != "ExtensibleVirtualDisk" {
```

Comment:
No longer needed for evd? Don't know the internals

Reply:
Yeah, this doesn't really make sense for EVD because EVD is not an actual file on the host (at least storage spaces are not) so this call doesn't make sense for EVD type.